src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -95,10 +95,14 @@ def __setattr__(self, name, value):
 
 class LongRunningOperation(object):  # pylint: disable=too-few-public-methods
 
-    def __init__(self, start_msg='', finish_msg='', poller_done_interval_ms=1000.0):
+    def __init__(self, start_msg='', finish_msg='',
+                 poller_done_interval_ms=1000.0):
         self.start_msg = start_msg
         self.finish_msg = finish_msg
         self.poller_done_interval_ms = poller_done_interval_ms
+        self.controller = ProgressHook(progress_type=ProgressType.Indeterminate)
+        from azure.cli.core.application import APPLICATION
```

Comment:
@oakeyc @derekbekoe suggests move it to the head of the method, not the file, does it will cause a cyclic imports issue?